src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -5869,6 +5869,45 @@ def test_aks_nodepool_drain_timeout(self, resource_group, resource_group_locatio
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
```

Comment:
TODO once Swagger change (https://github.com/Azure/azure-rest-api-specs/pull/26762) merged & SDK updated, queue live test and add recording. See earlier discussion on preview CLI with @FumingZhang [here](https://github.com/Azure/azure-cli-extensions/pull/6865/files#r1364821249)

Reply:
Swagger change now merged, waiting for SDK update

Reply:
Looks like SDK release will happen shortly

https://github.com/Azure/azure-sdk-for-go/pull/22271

Reply:
Live test: https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=86189118&view=results